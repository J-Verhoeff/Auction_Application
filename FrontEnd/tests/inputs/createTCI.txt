login
admin
create
admin
create
logout
create
usernameover15characters
create
user1
full
create
user1
full-standard
-1
create
user1
full-standard
1000000
create
user1
full-standard
10a
create
user1
full-standard
10.0.0
create
user1
full-standard
10.000
create
user1
full-standard
10.10
logout
exit