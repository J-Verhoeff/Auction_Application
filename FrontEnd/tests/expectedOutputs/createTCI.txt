
Welcome to the Auction App!
Start entering transactions
To exit the application type 'exit'

Options:
	login
Transaction: User: 
Welcome admin!
User Type: AA
Available Credit: 100.00$

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: New username: New username cannot be an already existing username!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: New username: New username cannot be a transaction code!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: New username: New username cannot be over 15 characters!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: New username: Type of user (admin, full-standard, buy-standard, sell-standard): Type of user cannot be other than the options listed!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: New username: Type of user (admin, full-standard, buy-standard, sell-standard): Credit amount: Credit amount must be a number between 0 and 999,999.99!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: New username: Type of user (admin, full-standard, buy-standard, sell-standard): Credit amount: Credit amount must be a number between 0 and 999,999.99!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: New username: Type of user (admin, full-standard, buy-standard, sell-standard): Credit amount: Credit amount must be a number between 0 and 999,999.99!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: New username: Type of user (admin, full-standard, buy-standard, sell-standard): Credit amount: Credit amount must be a number between 0 and 999,999.99!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: New username: Type of user (admin, full-standard, buy-standard, sell-standard): Credit amount: Credit amount can only have 2 decimal places!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: New username: Type of user (admin, full-standard, buy-standard, sell-standard): Credit amount: New user successfully created!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: 
Welcome to the Auction App!
Start entering transactions
To exit the application type 'exit'

Options:
	login
Transaction: Goodbye!
